config/routes.rb

```diff
@@ -18,6 +18,9 @@
   get "/bo/convictions/approved" => "convictions_dashboards#approved", as: :convictions_approved
   get "/bo/convictions/rejected" => "convictions_dashboards#rejected", as: :convictions_rejected
 
+  # Privacy policy
+  get "/ad-privacy-policy/:reg_identifier", to: "ad_privacy_policy#show", as: :ad_privacy_policy
```

Comment:
```suggestion
  get "/bo/ad-privacy-policy/:reg_identifier", to: "ad_privacy_policy#show", as: :ad_privacy_policy
```

We have to include /bo at the start of all back office routes. This is because the backend and the back office are on the same subdomain and we need to check the URL to decide which app to route to.

Think this will require an update to https://github.com/DEFRA/waste-carriers-frontend as well.

Reply:
Oh!! will look into that! Locally it worked fine 🤔

Reply:
Yeah, locally we have different ports for the different apps so it's not an issue. But on production it would be.